static_src/test/unit/actions/service_actions.spec.js

```diff
@@ -17,17 +18,40 @@ describe('serviceActions', function() {
     sandbox.restore();
   });
 
+  function setupViewSpy() {
+    return sandbox.spy(AppDispatcher, 'handleViewAction');
```

Comment:
if you find yourself referring to the string `'handleViewAction'` or `'handleServerAction'` elsewhere in the code, I would make those constants. i'm not sure if you do use them more than here. i remember i did that in the angular stuff and wished i didn't. too many places to find and replace when refactoring. your call on this one.